test/e2e/app-dir/x-forwarded-headers/x-forwarded-headers.test.ts

```diff
@@ -63,5 +63,16 @@ createNextDescribe('x-forwarded-headers', { files: __dirname }, ({ next }) => {
       expect(headers['middleware-x-forwarded-port']).toBe(reqHeaders.port)
       expect(headers['middleware-x-forwarded-proto']).toBe(reqHeaders.proto)
     })
+
+    it.only('should work with multiple x-forwarded-* headers', async () => {
+      const res = await next.fetch('/', {
+        headers: { 'x-forwarded-proto': 'https, https' },
+      })
+
+      expect(res.status).toBe(200)
+
+      const headers = await res.json()
+      expect(headers['x-forwarded-proto']).toBe('https, https')
```

Comment:
@balazsorban44  Just confirming if this value should be just **'https'** or **'https, https'**

Reply:
I assume we don't want to modify the original header value, we just want to set the correct protocol for the server.

The behavior is not new in this PR, it just now supports passing a concatenated value as `x-forwarded-proto`

Reply:
Thanks for the explanation 👍